app/src/test/kotlin/maru/app/MaruQbftTest.kt

```diff
@@ -48,7 +48,9 @@ import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.io.TempDir
+import org.junit.jupiter.api.parallel.Isolated
 
+@Isolated
```

Comment:
While this brings value short term to make the CI more stable, I think we should address it anyway and maybe create an issue to make it parallel once we have time

Reply:
Please use issue templates we have in the future